CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 Version History
 ---------------
 
+May 2017:
+
+v1.2.2 (patch release)
+- Fix seed-related bugs affecting wallet balance
+- Gracefully handle missing storage folders
+- Faster + smaller wallet database
```

Comment:
diff shows 2600 lines of changes since v1.2.1 in files matching `*_test.go`. 3200 lines total. So maybe "2500 lines of new testing"